win32/Lib/win32serviceutil.py

```diff
@@ -6,7 +6,7 @@
 # when things go wrong - eg, not enough permissions to hit the
 # registry etc.
 
-import importlib
+import importlib.machinery
```

Comment:
See https://github.com/python/typeshed/pull/10746#issuecomment-1730371768 as to why importing directly from `importlib` shouldn't be relied on for `importlib.machinery`, `importlib.util` and `importlib.abc`. Mypy & Pyright (see #2102) will also give an error on this.